server/routes/visitJourney/selectVisitors.ts

```diff
@@ -96,6 +98,16 @@ export default class SelectVisitors {
       restriction => restriction.restrictionType === 'CLOSED',
     )
 
+    const allSelectedVisitorBans: Restriction[] = selectedVisitors.reduce((acc, visitor) => {
+      visitor.restrictions.forEach(restriction => {
+        acc.push(restriction)
+      })
+      return acc
+    }, [])
+
```

Comment:
Much simpler solution:
```suggestion
const allSelectedVisitorBans = selectedVisitors.flatMap(visitor => visitor.restrictions)
```
apologies - I think I overcomplicated it in a previous comment...